R/pkg/R/DataFrame.R

```diff
@@ -945,6 +947,19 @@ setMethod("[", signature(x = "DataFrame", i = "missing"),
             select(x, j)
           })
 
+#' @rdname select
+setMethod("[", signature(x = "DataFrame", i = "Column"),
+          function(x, i, j, ...) {
```

Comment:
... seems not necessary? It allows user pass parameters that won't be used, which may be confusing.